test/minify.test.ts

```diff
@@ -636,7 +636,7 @@ describe('Webpack 5', () => {
 		expect(asset.info.minimized).toBe(true);
 
 		const file = built.fs.readFileSync('/dist/test.js', 'utf-8');
-		expect(file).toBe('const e=1;export default e;\n');
+		expect(file).toBe('const e=1;export default 1;\n');
```

Comment:
This doesn't look right. Why does it export `1` instead of `e`?

Reply:
It appears that put `const e=1; export default e;` or `export default 1;` in `a.mjs`, and `import e from "./a.mjs"; e = 2;` in `b.mjs` leads to a same result.

However, I'm not familiar with ES module import/export. `git bisect` shows this commit causes the change: evanw/esbuild@7bf3046f723f4c76b3c61ce9d2724468150c1722. Do you think it's a bug from `esbuild`?

Reply:
Yeah I suppose this is technically a safe change because exports are immutable.

I'm wondering why it preserves the `const e` though. Given it's an ESM module, it should be safe to remove the const.

I'll take a closer look in the morning but I think the changes are safe to release.

Thanks for investigating!

---

Wow, how did you use git bisect on a npm dependency? Did you symlink and re-build esbuild on every commit?

Reply:
Hi, do you have any update on this?

---

I simply bisected esbuild and ran the binary with input `const value = 1;\n\nexport default value;` :smile_cat:

Reply:
Thanks for the reminder 😅

Looks good!